PWGHF/Tasks/taskPidStudies.cxx

```diff
@@ -144,6 +145,7 @@ struct HfTaskPidStudies {
   Configurable<float> massLambdaMax{"massLambdaMax", 1.3, "Maximum mass for lambda"};
   Configurable<float> massOmegaMin{"massOmegaMin", 1.5, "Minimum mass for omega"};
   Configurable<float> massOmegaMax{"massOmegaMax", 1.8, "Maximum mass for omega"};
+  Configurable<float> maxRadius{"maxRadius", 2.3, "Maximum decay radius (cm)"};
```

Comment:
```suggestion
  Configurable<float> radiusMax{"radiusMax", 2.3, "Maximum decay radius (cm)"};
```
Please follow the `<object><property>` naming convention.

Reply:
Hi @vkucera, thanks! I have implemented it